### PR TITLE
IND-113 - Refactor the Corporate Actions ontology to create a standalone actions ontology that codes can be mapped to

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

Moved the most general concepts that can apply to a legal entity or at least at the business level out of the security-related corporate actions ontology and into a general corporate actions ontology to facilitate further refactoring

Fixes: #1650 / IND-113

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](https://github.com/edmcouncil/fibo/blob/master/CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

**Review by tahoeblue — DISMISSED** [review]

**Comment by rivettp** [comment]
OK for now

File: CAE/CorporateEvents/CorporateActions.rdf [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+		<rdfs:subClassOf rdf:resource="&lcc-lr;CodeSet"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;hasMember"/>
+				<owl:allValuesFrom rdf:resource="&fibo-cae-ce-act;ActionClassifier"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;defines"/>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/CorporateActions.rdf:102]: In our classification schemes ontology, the scheme defines the classifier.

File: CAE/CorporateEvents/CorporateActions.rdf [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+		<skos:definition>scheme for classifying the kinds of actions and events that may be announced, initiated or carried out by an organization that affects a legal entity or the securities it issues</skos:definition>
+		<fibo-fnd-utl-av:usageNote>The set of corporate actions and income events included herein are a subset of those specified in a combination of ISO 15022 Securities - Scheme for Messages (Data Field Dictionary) and the GLEIF LEI-related corporate actions. Other schemes that are specific to a custodian, depository, or regulatory agency may also be important, and should take a similar approach with respect to classification.</fibo-fnd-utl-av:usageNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-cae-ce-act;ActionClassifier">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-oc;OccurrenceKind"/>
+		<rdfs:subClassOf rdf:resource="&lcc-lr;CodeElement"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isDefinedIn"/>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/CorporateActions.rdf:116]: The scheme does define the classifiers that are part of a scheme though.

File: CAE/CorporateEvents/CorporateActions.rdf [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-cr;classifies"/>
+				<owl:someValuesFrom rdf:resource="&fibo-cae-ce-act;Action"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;denotes"/>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/CorporateActions.rdf:136]: Removed, although that is what a code element does.  Given that we have changed this from CodeElement to Classifier, that's moot.

File: CAE/CorporateEvents/CorporateActions.rdf [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>action classification scheme</rdfs:label>
+		<skos:definition>scheme for classifying the kinds of actions and events that may be announced, initiated or carried out by an organization that affects a legal entity or the securities it issues</skos:definition>
+		<fibo-fnd-utl-av:usageNote>The set of corporate actions and income events included herein are a subset of those specified in a combination of ISO 15022 Securities - Scheme for Messages (Data Field Dictionary) and the GLEIF LEI-related corporate actions. Other schemes that are specific to a custodian, depository, or regulatory agency may also be important, and should take a similar approach with respect to classification.</fibo-fnd-utl-av:usageNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-cae-ce-act;ActionClassifier">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-oc;OccurrenceKind"/>
+		<rdfs:subClassOf rdf:resource="&lcc-lr;CodeElement"/>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/CorporateActions.rdf:113]: Yup - changed to Classifier

File: CAE/CorporateEvents/CorporateActions.rdf [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+				<owl:someValuesFrom rdf:resource="&xsd;string"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">action classifier</rdfs:label>
+		<skos:definition xml:lang="en">classifier that distinguishes the kinds of actions and events that may be announced, initiated or carried out by an organization that affects a legal entity or the securities it issues, such as income-oriented events</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">ISO 15022 classifies events as impacting income vs. others. Other classification schemes distinguish between actions that return profits to shareholders, actions that are designed to influence the share price, and actions involving a change in structure to the issuer organization.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-cae-ce-act;BusinessStrategyOrientedAction">
+		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;ActionClassifier"/>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/CorporateActions.rdf:152]: Moved to security-specific -- this is part of the ISO 15022 code scheme - they are organized by whether they are business strategy oriented or income oriented.

File: CAE/CorporateEvents/CorporateActions.rdf [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+		<skos:definition xml:lang="en">The issuing company changes it&apos;s name. Event shows the change from old name to new name and may involve surrendering physical shares with the old name to the registrar. SWIFT = NAME</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-cae-ce-act;PlaceOfIncorporationChange">
+		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;CorporateInformationAction"/>
+		<rdfs:label xml:lang="en">place of incorporation change</rdfs:label>
+		<skos:definition xml:lang="en">Changes in the state of incorporation for US companies and changes in the place of incorporation for foreign companies. Where shares need to be registered following the incorporation change, the holder(s) may have to elect the registrar. SWIFT = PLAC</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-cae-ce-act;PriorityIssue">
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/CorporateActions.rdf:262]: Ok - noted, but will have to see how it is used to figure that out in a follow-up version of this.

File: CAE/CorporateEvents/CorporateActions.rdf [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+		<owl:imports rdf:resource="https://www.omg.org/spec/LCC/Languages/LanguageRepresentation/"/>
+		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/CAE/CorporateEvents/CorporateActions/"/>
+		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Provisional"/>
+	</owl:Ontology>
+	
+	<owl:Class rdf:about="&fibo-cae-ce-act;Action">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-oc;Occurrence"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;appliesTo"/>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply CAE/CorporateEvents/CorporateActions.rdf:66]: So - it inherits a date and time ... let's add this in the next iteration.

**Review by jfgemski — DISMISSED** [review]

**Review by jfgemski — APPROVED** [review]

**Review by rivettp — APPROVED** [review]

**Pull request merged**